src/plots/plots.js

```diff
@@ -1714,6 +1722,9 @@ plots.transition = function(gd, data, layout, traces, frameOpts, transitionOpts)
     var aborted = false;
 
     function executeTransitions() {
+
+        gd.emit('plotly_transitioning', []);
```

Comment:
Subtle, but I didn't like that this was executed asynchronously with the transition actually starting, hence the move.